mev_inspect/aave_liquidations.py

```diff
@@ -0,0 +1,105 @@
+from typing import List, Optional, Dict
+
+
+from mev_inspect.schemas.classified_traces import (
+    ClassifiedTrace,
+    DecodedCallTrace,
+    Classification,
+    Protocol,
+)
+
+from mev_inspect.schemas.liquidations import Liquidation
+
+AAVE_CONTRACT_ADDRESSES: List[str] = [
+    # AAVE Proxy
+    "0x398ec7346dcd622edc5ae82352f02be94c62d119",
+    # AAVE V2
+    "0x7d2768de32b0b80b7a3454c06bdac94a69ddc7a9",
+    # AAVE V1
+    "0x3dfd23a6c5e8bbcfc9581d2e864a68feb6a076d3",
+    # AAVE V2 WETH
+    "0x030ba81f1c18d280636f32af80b9aad02cf0854e",
+]
+
+
+def is_transfer_to_liquidator(
+    trace: ClassifiedTrace, liquidator: str
+) -> Optional[ClassifiedTrace]:
+    """Check if transfer is to liquidator"""
+
+    if isinstance(trace, DecodedCallTrace):
+        try:
```

Comment:
try to limit `try` except usage unless truly there's some sort of error (ex: a call to an API times out). better to have explicit control flow

Example:
```
if "recipient" in trace.inputs:
    recipient = trace.inputs["reciplient"]
    if recipient == liquidator and trace.from_address in AAVE_CONTRACT_ADDRESSES:
        return trace

if "dst" in ...
```

Reply:
Got it